src/lib.rs

```diff
@@ -44,6 +45,34 @@ pub fn guess(xdr_base64: String) -> Vec<String> {
         .collect()
 }
 
+/// Decodes the XDR into JSON.
+///
+/// Accepts a XDR base64 string.
+///
+/// Returns a JSON string.
+///
+/// Unstable: The API of this function is unstable and will likely be changed to
+/// return a JsValue instead of a JSON string.
```

Comment:
I think we can consider the API stable for this major version of the library, and at the moment in time we explore JsValue (I explored it a bit fwiw, and ran into some hurdles), we can consider how to make the breaking change.
```suggestion
```